tests/queries/0_stateless/02884_string_distance_function.sql

```diff
@@ -52,4 +52,13 @@ select levenshteinDistance(s1, s2) FROM t ORDER BY s1, s2;
 
 SELECT editDistance(randomString(power(2, 17)), 'abc'); -- { serverError TOO_LARGE_STRING_SIZE}
 
+select 'jaroSimiliary';
+select jaroSimilarity('duck', 'duckdb');
+
+select 'dameerauLevenshteinDistance';
+select damerauLevenshteinDistance('duckdb', 'udckbd');
+
+select 'jaroWinklerSimilarity';
+select jaroWinklerSimilarity('duck', 'duckdb');
```

Comment:
I would like to see more tests with various edge cases (one/two empty strings, values that above `boost_threshold`, values that are below, etc).

And it is the tiniest thing, but I guess we can have better tests cases than the name of another OLAP database product :smile: